web/src/components/SortButton/SortButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectSeparator,
+  SelectTrigger,
+  SelectValue,
+} from '../ui/select'
+
+const SortButton = () => {
+  return (
+    <Select>
+      <SelectTrigger className="w-[210px]">
+        <SelectValue placeholder="Sorteren" />
+      </SelectTrigger>
+      <SelectContent>
+        <SelectGroup>
+          <SelectLabel>Sorteren op</SelectLabel>
+          <SelectSeparator />
+          <SelectItem value="latest">Laatste</SelectItem>
+          <SelectItem value="oldest">Oudste</SelectItem>
+          <SelectItem value="recentlyUpdated">Onlangs bijgewerkt</SelectItem>
+          <SelectItem value="leastRecentlyUpdated">
```

Comment:
```suggestion
          <SelectItem value="mostStaleFirst">
```

Is "stale" a bit cheeky?hehe

Reply:
stale does sound like a diss haha